superset/models/core.py

```diff
@@ -87,6 +87,18 @@ class KeyValue(Model):  # pylint: disable=too-few-public-methods
     value = Column(Text, nullable=False)
 
 
+class CacheKey(Model):  # pylint: disable=too-few-public-methods
+
+    """Used for any type of key-value store"""
```

Comment:
Let's make the description a bit more accurate - I would have thought this model was itself a key/value store, rather than saved references to keys in a different cache back-end.